.gitlab-ci.yml

```diff
@@ -1017,6 +1017,7 @@ stages:
     JANE_INVENTORY_HOSTVARS: 'roundcube__database=mysql-default'
     JANE_DIFF_PATTERN: '.*/debops.roundcube/.*'
     JANE_LOG_PATTERN: '\[debops\.roundcube\]'
+  tags: [ 'mem-6GB' ]
```

Comment:
Great. I just forgot about [setting the Vagrant box memory](https://github.com/debops/debops/blob/master/.gitlab-ci.yml#L450) sorry about that. Can you add the
```yaml
  VAGRANT_MASTER_MEMORY: '4096'
```
variable as well?